swap/src/protocol/alice/swap.rs

```diff
@@ -45,6 +45,8 @@ where
             .await?;
     }
 
+    swap.monero_wallet.store().await?;
```

Comment:
Quite arbitrary to store the wallet at this point. What if the ASB does not complete a swap for a whole week? Then the wallet will never be stored

Reply:
I think we should do this periodically. Scanning blocks, regardless of whether or not they contain any transactions that affect our balance, takes time and storing the wallet skips scanning those empty blocks for the next time. 


I'm quite confused why the monero-wallet-rpc doesn't do this itself. I'd expect the wallet to be stored periodically...

Reply:
Quote from a Monero core dev

<moneromooo> It saves at (non fatal) exit. Save it yourself when you know it's important. Saving can take some time.

Reply:
Yeah periodically sounds good then